components/account/AccountAvatar.vue

```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import type { mastodon } from 'masto'
 
-const { account, square } = defineProps<{
+const props = defineProps<{
```

Comment:
@AndrewCrescencio Thanks for the fix!

About the object destructuring, that warning was valid in Vue 3.4 (the previous result of `pnpm run test:ci`), but Vue 3.5+ allows us to use the same syntax again. So we can still use the original destructuring on `props`. Could you revert that destructuring since we already used it in many other places if you don't mind? You can also check by running `pnpm run test:ci` on your environment.

These links will help to understand the latest (Vue 3.5+) usage of props destructuring:
- Props | Vue.js - https://vuejs.org/guide/components/props#reactive-props-destructure
- Vue 3.5+: How to Destructure Props in 2024 (Composition API) - https://www.docs4.dev/posts/vue-3-how-to-destructure-props-in-2024

I've updated the test result with the latest commit, that doesn't contains the previous warning: https://github.com/elk-zone/elk/issues/2934

Reply:
Nice that we still can use props destructure, thanks for info